biomage/utils/data.py

```diff
@@ -83,12 +83,6 @@ def copy_s3_files(sandbox_id, prefix, source_bucket, target_bucket):
         experiment_id = obj["Key"].split("/")[0]
         target_key = obj["Key"].replace(experiment_id, f"{sandbox_id}-{experiment_id}")
 
-        # biomage-originals- uses projectId/sampleId/file schema,
-        # so sampleId has to be prefixed too
-        if 'biomage-originals-' in target_bucket:
-            sample_id = obj["Key"].split("/")[1]
-            target_key = target_key.replace(sample_id, f"{sandbox_id}-{sample_id}")
-
```

Comment:
Why was this deleted?

Reply:
The deleted line of code prefixes the sample ids in the S3 sample file path (schema : projectUuid/sampleUuid/filename) when copying files. However, when copying the experiment data, we do not prefix sample ids. We do not prefix sampleIds because we also copy the user's processed Seurat object, which contains unprefixed sample object. If we prefix sample-ids, we will have problem accessing the data in the Seurat object.